<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RvConsoleDialog</class>
 <widget class="QDialog" name="RvConsoleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>336</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="clearButton">
       <property name="text">
        <string>Clear Display   </string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="biggerButton">
       <property name="text">
        <string>Bigger</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="smallerButton">
       <property name="text">
        <string>Smaller</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Show on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="showComboBox">
       <item>
        <property name="text">
         <string>Error</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Error or Warning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Error, Warning, or Info</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Any Output</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Manually</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources>
  <include location="RvCommon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RvConsoleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>biggerButton</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smallerButton</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>7</y>
    </hint>
    <hint type="destinationlabel">
     <x>481</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
